ocs_ci/utility/version.py

```
@@ -73,6 +73,7 @@ def get_semantic_version(
 VERSION_4_20 = get_semantic_version("4.20", True)
 VERSION_4_21 = get_semantic_version("4.21", True)
 VERSION_4_22 = get_semantic_version("4.22", True)
+VERSION_5_0 = get_semantic_version("5.0", True)
```

Ruff (0.15.17)

[warning] Boolean positional value in function call

(FBT003)